src/types/Intent.ts

```diff
@@ -1,6 +1,7 @@
 export type Intent =
   | 'default'
   | 'primary'
+  | 'standard'
   | 'secondary'
   | 'info'
   | 'success'
```

Comment:
Here as well. We can keep the old `Intent` type because it used in other components, but I think making a new type called `Variant` is better moving forward.